spyder/widgets/fileswitcher.py

```diff
@@ -652,7 +658,7 @@ def setup_file_list(self, filter_text, current_path):
             self.set_current_row(self.filtered_path.index(current_path))
         elif self.filtered_path:
             self.set_current_row(0)
-        self.fix_size(short_paths, extra=200)
+        self.fix_size(short_paths, extra=100)
```

Comment:
Since the list for the symbols normally has less width, I was thinking that would be better to reduce the width given by the `extra` param.